infra/aws/terraform/modules/eks-prow-iam/policies.tf

```diff
@@ -14,9 +14,15 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+resource "aws_iam_policy" "eks_plan" {
```

Comment:
Why's there no `name` attribute for this resource?

Reply:
Terraform can autogenerate a name.
Might be good to set `name_prefix` though.

> If omitted, Terraform will assign a random, unique name.

Reply:
I've added an explicit name here.